testing/src/test/java/com/github/bsideup/liiklus/container/LiiklusContainerTest.java

```diff
@@ -0,0 +1,26 @@
+package com.github.bsideup.liiklus.container;
+
+import org.junit.Test;
+import org.testcontainers.containers.KafkaContainer;
+
+import static org.junit.Assert.fail;
+
+public class LiiklusContainerTest {
+
+    static KafkaContainer kafka = new KafkaContainer();
+
+    static {
+        kafka.start();
+    }
+
+    @Test
+    public void shouldStart() {
+        try (LiiklusContainer liiklusContainer = new LiiklusContainer("0.4.5")) {
+
+            liiklusContainer.withKafka(kafka).start();
+
+        } catch (Exception e) {
```

Comment:
why this catch? :) JUnit will fail the test if there is an exception

Reply:
I and @lanwen had discussion. I think it is because he mean to explicitly indicate a test is fail in Junit way :)